build/concurrent_jobs.gni

```diff
@@ -0,0 +1,12 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+_script = "//flutter/build/get_concurrent_jobs.py"
+_args = [
+  "--reserve-memory=1GB",
+  "--memory-per-job",
+  "dart=1GB",
```

Comment:
I was trying to figure out why memory-per-job was a key value pair but it looks like you are going to extend this for other things as well. Perhaps something for the linker when LTO is enabled maybe?

Reply:
Yeah, the Chromium and Fuchsia builds use this in LTO builds. We should probably consider doing that as well after profiling.